aspnetcore/signalr/configuration.md

```diff
@@ -62,6 +62,7 @@ The following table describes options for configuring SignalR hubs:
 | `KeepAliveInterval` | 15 seconds | If the server hasn't sent a message within this interval, a ping message is sent automatically to keep the connection open. When changing `KeepAliveInterval`, change the `ServerTimeout`/`serverTimeoutInMilliseconds` setting on the client. The recommended `ServerTimeout`/`serverTimeoutInMilliseconds` value is double the `KeepAliveInterval` value.  |
 | `SupportedProtocols` | All installed protocols | Protocols supported by this hub. By default, all protocols registered on the server are allowed, but protocols can be removed from this list to disable specific protocols for individual hubs. |
 | `EnableDetailedErrors` | `false` | If `true`, detailed exception messages are returned to clients when an exception is thrown in a Hub method. The default is `false`, as these exception messages can contain sensitive information. |
+| `StreamBufferCapacity ` | `10` | The maximum buffer size for client upload streams. If this limit is reached the processing of invocations will be blocked until the server processes stream items.|
```

Comment:
```suggestion
| `StreamBufferCapacity` | `10` | The maximum buffer size for client upload streams. If this limit is reached, the processing of invocations is blocked until the server processes stream items.|
```

Reply:
Sounds to me like this property was introduced in 3.0. If that's the case, we'll need to maintain 2 different tables: one for 2.2 and below and another for 3.0+.

Reply:
Using the moniker range keywords? Like
 `::: moniker range="= aspnetcore-3.0"`
    Content
`::: moniker-end`